07-systems_access_policy.md

```diff
@@ -177,14 +177,6 @@ Google Workspace password requirements shall be set to “Enforce strong passwor
 * Password reuse not allowed
 * 2FA required
 
-MongoDB password requirements shall be set to the strongest system settings of the following:
```

Comment:
Do we also want to add postman to this list since some users can get ePHI?

OR, should we turn on `GSuite` and turn off `Postman password` 

<img width="825" alt="image" src="https://github.com/healthnoteinc/policies/assets/3267978/5fdb3482-1f86-459f-b331-ea46d06c00e9">

Reply:
I believe MongoDB did not have configurable settings but we documented the strongest settings possible.  That is why it is listed there.

It looks like Postman doesn't have configurable settings either and has a minimum of 7 characters.  

GSuite is the full SAML SSO setup, which I thought we had setup, but may have turned it off in favor of using Google Oauth 2.0 instead to simplify usage (Login with Google option).